dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/MergeConfigServerXMLTest.java

```diff
@@ -0,0 +1,290 @@
+
+        assertEqualIgnoringPropertyOrder(expectedInfo, openapiNode.path("info"));
+
+        // Now update the info and check the change is reflected
+        info.setVersion("3.8");
+        setMergeConfig(list("all"), null, info);
+
+        doc = OpenAPIConnection.openAPIDocsConnection(server, false).download();
+        openapiNode = OpenAPITestUtil.readYamlTree(doc);
+
+        expectedInfo.put("version", "3.8");
+
+        assertEqualIgnoringPropertyOrder(expectedInfo, openapiNode.path("info"));
+    }
+
+    @Test
+    public void testInfoConfigured31() throws Exception {
+        MpOpenAPIInfoElement info = new MpOpenAPIInfoElement();
+        info.setTitle("test title");
+        info.setVersion("3.7");
+        info.setTermsOfService("http://example.org/tos");
+        info.setContactName("John Smith");
+        info.setContactUrl("http://example.org/contact");
+        info.setLicenseName("Apache 2.0");
+        info.setLicenseUrl("https://www.apache.org/licenses/LICENSE-2.0.html");
+        info.setDescription("This is a test API to test configuration of the info object");
+        info.setLicenseIdentifier("Apache-2.0");
+        info.setSummary("A test API");
+
+        setMergeConfig(list("all"), null, info);
+
+        WebArchive war1 = ShrinkWrap.create(WebArchive.class, "test1.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource.class);
+        deployApp(war1);
+
+        WebArchive war2 = ShrinkWrap.create(WebArchive.class, "test2.war")
+                                    .addClasses(DeploymentTestApp.class, DeploymentTestResource2.class);
+        deployApp(war2);
+
+        String doc = OpenAPIConnection.openAPIDocsConnection(server, false).download();
+        JsonNode openapiNode = OpenAPITestUtil.readYamlTree(doc);
+
+        ObjectNode expectedInfo = JsonNodeFactory.instance.objectNode();
+        expectedInfo.put("title", "test title");
+        expectedInfo.put("version", "3.7");
+        expectedInfo.put("termsOfService", "http://example.org/tos");
+        expectedInfo.put("description", "This is a test API to test configuration of the info object");
+
+        ObjectNode contact = expectedInfo.putObject("contact");
+        contact.put("name", "John Smith");
+        contact.put("url", "http://example.org/contact");
+
+        ObjectNode license = expectedInfo.putObject("license");
+        license.put("name", "Apache 2.0");
+        license.put("url", "https://www.apache.org/licenses/LICENSE-2.0.html");
+
+        if (OpenAPITestUtil.getOpenAPIFeatureVersion() >= 4.0f) {
+            // Additional fields only available in OpenAPI 3.1
+            expectedInfo.put("summary", "A test API");
+            license.put("identifier", "Apache-2.0");
+        }
+
+        assertEqualIgnoringPropertyOrder(expectedInfo, openapiNode.path("info"));
+    }
+
+    @Test
+    public void testInfoNoTitle() throws Exception {
```

Comment:
I think we should have a noVersion test too. And maybe even an empty title/version test.

Reply:
I've added a no version test.